language-support/scala/codegen-sample-app/src/test/scala/com/digitalasset/codegen/util/TestUtil.scala

```diff
@@ -2,14 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0
 
 package com.digitalasset.codegen.util
+
+import com.digitalasset.daml.bazeltools.BazelRunfiles
```

Comment:
```suggestion
import com.digitalasset.daml.bazeltools.BazelRunfiles._
```

with

```scala
object BazelRunfiles extends BazelRunfiles
```

added to `BazelRunfiles.scala`.

Reply:
Thanks

Reply:
@S11001001 PTAL